aws/resource_aws_lb_test.go

```diff
@@ -1026,7 +1139,7 @@ resource "aws_lb" "test" {
 }
 
 resource "aws_eip" "lb" {
-  count = "${length(data.aws_availability_zones.available.names)}"
+  count = "${length(data.aws_availability_zones.available.names)>2?2:length(data.aws_availability_zones.available.names)}"
```

Comment:
Are there any regions with only 1 AZ? Seems like this should just be `count = 2`